styles/globals.css

```diff
@@ -54,8 +61,4 @@ a {
   #overlay {
     display: block;
   }
-}
-
-
-@import url('https://fonts.cdnfonts.com/css/iceberg');
```

Comment:
Should probably leave the font

Reply:
We still do have that font. I just imported together with the other one. line 4 in styles/globals.css